package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "koordinattransformation",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "@dataforsyningen/designsystem": "^8.1.1",
+    "ol": "^10.2.1",
+    "proj4": "^2.14.0",
+    "vue": "^3.5.12",
+    "vue-router": "^4.4.5",
+    "vuex": "^4.1.0"
```

Comment:
Er vuex stadig anvendelig med Vue3? Jeg havde indtryk af, at den "officielle" state manager for Vue3 var Pinia (https://pinia.vuejs.org/ )

Reply:
Jeg får taget et kig på det :) Tak for info

Reply:
Har fået migreret til Pinia